maquetado/scss/_navbar.scss

```diff
@@ -0,0 +1,27 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 1px 10px 8px $grey;
+  display: flex;
+  height: $title-decoration-height;
+  overflow: hidden;
+  padding: 15px;
+  position: fixed;
+  text-align: end;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+
+  &::before {
```

Comment:
Very good :+1:  but in this case if is necessary that you put the image tag in the HTML